js/apps/system/_admin/aardvark/APP/frontend/js/arango/arango.js

```diff
@@ -1200,20 +1207,32 @@
         )
       );
 
+      if (window.App.replaceApp) {
+        tableContent.push(
+          window.modalView.createCheckboxEntry(
+            'new-app-flag-teardown',
+            'Run teardown?',
+            true,
+            "Should the existing service's teardown script be executed before replacing the service?",
```

Comment:
We didn't actually ask about the teardown when replacing an app. This used to be implied by the replace/upgrade flag.